config/environments/development.rb

```diff
@@ -38,4 +38,18 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+   # mailcatcher config
+   # to view emails generated in development:
+   # `gem install mailcatcher && mailcatcher`
+   config.action_mailer.delivery_method = :smtp
+
+   config.action_mailer.smtp_settings = {
+    address: "localhost",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.